docs/src/main/sphinx/release/release-358.md

```diff
@@ -0,0 +1,39 @@
+# Release 358 (XX May 2021)
+
+## General
+
+* Support any query in {doc}`/sql/show-stats`. ({issue}`8026`)
+* Improve performance of complex queries involving joins and `TABLESAMPLE`. ({issue}`8094`)
+* Improve performance of `ORDER BY ... LIMIT` queries on sorted data. ({issue}`6634`)
+* Reduce worker graceful shutdown time. ({issue}`8149`)
+* Fix query failure when one of join build side's columns is of non-orderable type. ({issue}`7723`)
+* Fix failure for queries containing repeated ordinals in `GROUP BY` clause.
+  Example: `SELECT x FROM t GROUP BY 1, 1`. ({issue}`8023`)
+* Fix failure for queries containing repeated expressions in `ORDER BY` clause of an aggregate function.
+  Example: `SELECT count(x ORDER BY y, y) FROM (VALUES ('a', 2)) t(x, y)`. ({issue}`8080`)  
+  
+## Security
+
+* Fix supports for multiple LDAP user bind patterns.  ({issue}`8022`)
```

Comment:
@dain, what exactly does this fix? Or is it more like "Add support for ..."?

Reply:
This was reverted - the correct fix is part of https://github.com/trinodb/trino/pull/8134

Reply:
Thanks, I'll remove the entry.